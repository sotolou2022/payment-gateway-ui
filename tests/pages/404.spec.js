import { mount, enableAutoUnmount } from "@vue/test-utils";
import { beforeEach, afterEach, expect } from "@jest/globals";
import Component from "~~/404";

let wrapper;

beforeEach(() => {
  wrapper = mount(Component);
});

enableAutoUnmount(afterEach);

describe("Test suite for 404 page", () => {
  it("is a vue component", () => expect(wrapper.vm).toBeTruthy());

  it("match snapshot", () => expect(wrapper.html()).toMatchSnapshot());
});
