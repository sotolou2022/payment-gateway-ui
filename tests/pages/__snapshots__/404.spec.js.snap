// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for 404 page match snapshot 1`] = `"Not Found"`;
